plenary/plenary-35.yaml

```diff
@@ -0,0 +1,325 @@
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to create ISO/TC 154/WG 7 with the title "Business"
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appointment of Mr. Klaus-Dieter Naujok as convener of ISO/TC 154/WG7"
+    identifier: 2016-11
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to appoint Mr. Klaus-Dieter Naujok as the convener of Working Group 7, effective immediately
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Allocation of the secretariat of ISO/TC 154/WG7 to SAC"
+    identifier: 2016-12
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves that the secretariat of ISO/TC 154/WG 7 is allocated to SAC. In addition the ISO/TC154 secretariat is requested to initiate an appropriate call for experts for the new WG 7
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Distribution of approved NWIP"
+    identifier: 2016-13
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: recommends
+        date_effective: 2016-10-21
+        message: recommends that all approved New Work Items Proposals from the MoU on eBusiness Management Group (MoU/MG) members shall be disseminated to the MoU/MG membership for review and possible action at the earliest stage
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Next ISO/TC 154 meetings"
+    identifier: 2016-14
+    approvals:
+      - type: affirmative
+        degree: # dissenting?
+        message: The resolution was approved with one negative vote by KATS
+
+    actions:
+      - type: agreed
+        date_effective: 2016-10-21
+        message: agreed to alternate its plenaries and working group meetings between physical and virtual meetings with the next meeting being a virtual meeting for the week starting 23 October 2017 and the next face-to-face meeting being held in 2018 in China
```

Comment:
In this case, I put `agreed` string value as `type`. Is it ok?

Reply:
Let's use the present tense for these actions, so "agrees" as type, but keep the "message".

Reply:
Ok, change applied.

Reply:
Thanks!